plugins/destination/mssql/client/write.go

```diff
@@ -0,0 +1,20 @@
+package client
+
+import (
+	"context"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/specs"
+)
+
+func (c *Client) WriteTableBatch(ctx context.Context, table *schema.Table, data [][]any) (err error) {
+	if len(data) == 0 {
```

Comment:
I think this is not needed as `WriteTableBatch` wont be called with no data

Reply:
removed